src/test/java/Homework20CH.java

```diff
@@ -0,0 +1,19 @@
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework20CH extends BaseTest{
+
+    @Test
+    public static void DeletePlaylist() {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example  `public static void deletePlaylist()`

Reply:
Thank you for the correction. I'll correct it.